src/keylessBackup/slice.ts

```diff
@@ -0,0 +1,41 @@
+import { PayloadAction, createSlice } from '@reduxjs/toolkit'
+
+export interface State {
+  google: {
+    loading: boolean
+    idToken: string | null
+    error: string | null
```

Comment:
nit: to avoid representing impossible states (both loading and error at the same time)
```suggestion
    status: 'idle' | 'loading' | 'success' | 'error'
    idToken: string | null
```